tools/notification-configuration/.gitignore

```diff
@@ -0,0 +1 @@
+launchSettings.json
```

Comment:
Is this necessary? I generally don't like having scoped .gitignore files.

Reply:
This is to prevent accidentally committing credentials set in Visual Studio. There are other `launchSettings.json` files scattered around this repo so scoping the .gitignore to this folder seemed appropriate... Equally appropriate would be an explicit entry in a repo-wide .gitignore.

Reply:
I'd just merge it up to the root ignore file.